tests/www_rbac/test_views.py

```diff
@@ -288,128 +292,200 @@ def prepare_dagruns(self):
             state=State.RUNNING)
 
     def test_index(self):
-        resp = self.client.get('/', follow_redirects=True)
+        resp = self.client.get('/', data=dict(
```

Comment:
there's really a huge amount of repetition here, I'd refactor the whole login part. Either a `get_as_test` method or better a `self.login(username='test')`. Tests would read:
```
self.login()  #maybe the default is 'tests' so no need to specify it
self.client.get('health')
```
I'd avoid the `follow_redirects` unless we're actually testing the redirect.

Reply:
hey Max, I do some update.